src/stores/AccountController.ts

```diff
@@ -450,17 +464,29 @@ function useLoginContext<T>(callback: (args: { cleengId: string; cleengSandbox:
   return useConfig((config) => callback({ ...config, customerId: user.id, auth }));
 }
 
+function useAccountContext<T>(callback: (args: { customerId: string; customer: Customer; auth: AuthData }) => T): T {
```

Comment:
This is needed because in some flows we don't need to make an API call, we just need the user data which is already in the store. Example: consents, capture..

Reply:
I would say just add Customer to the useLoginContext function callback. Otherwise these 2 are almost identical.

Reply:
Well, the `useLoginContext` uses `useConfig` where cleeng stuff exists and is used in a lot of places. If we try to modify the `useConfig` it will result in messy solution I think.

Reply:
OK this is hopefully all moderately temporary, so I won't worry about it.